src/ui/actions/layout.ts

```diff
@@ -63,6 +69,22 @@ export function setViewMode(viewMode: ViewMode): UIThunkAction {
       await dismissLocalNag(LocalNag.YANK_TO_SOURCE);
       dispatch(setSelectedPrimaryPanel("explorer"));
     }
+
+    let toggleTimeoutKey: NodeJS.Timeout | null = null;
+
+    //Update toggleMode to start toggle animation
+    dispatch(setToggleMode(viewMode));
+
+    // Delay updating the viewMode in redux so that the toggle can fully animate
+    // before re-rendering all of devtools in the new viewMode.
+    if (toggleTimeoutKey) {
+      clearTimeout(toggleTimeoutKey);
+    }
```

Comment:
This won't be toggled because `toggleTimeoutKey` here will always be undefined, since it doesn't get set until well after.